services/channel/service/package.json

```diff
@@ -0,0 +1,108 @@
+    "clean": "rimraf dist",
+    "build": "yarn clean && tsc --project tsconfig.build.json",
+    "start": "yarn util:load-vars node dist/index",
+    "dev": "concurrently -n \"tsc,graphile-migrate\" -c \"bgBlue.bold,bgMagenta.bold\" \"yarn dev:watch\" \"yarn internal:migrate-watch\"",
+    "dev:watch": "yarn util:load-vars tsc-watch --project tsconfig.build.json --onSuccess \"node dist/index.js\"",
+    "debug": "concurrently -n \"tsc,graphile-migrate\" -c \"bgBlue.bold,bgMagenta.bold\" \"yarn debug:watch\" \"yarn internal:migrate-watch\"",
+    "debug:watch": "yarn util:load-vars tsc-watch --project tsconfig.build.json --onSuccess \"node --inspect dist/index.js\"",
+    "test:reset:dbs": "yarn util:load-vars ts-node --files scripts/test-reset-dbs.ts",
+    "test": "yarn util:load-vars jest --silent",
+    "test:watch": "yarn test --watch",
+    "test:debug": "yarn util:load-vars node --inspect -r tsconfig-paths/register -r ts-node/register ../../../node_modules/jest/bin/jest --runInBand --silent",
+    "test:cov": "yarn util:load-vars jest --coverage --silent && ts-node ../../../scripts/open-test-coverage.ts -- services/channel/service",
+    "db:commit": "yarn util:load-vars ts-node --files scripts/db-commit.ts",
+    "db:uncommit": "yarn util:load-vars ts-node --files scripts/db-uncommit.ts",
+    "db:reset": "yarn util:load-vars ts-node --files scripts/db-reset.ts",
+    "db:update-schema": "yarn util:load-vars ts-node --files scripts/db-reset-shadow.ts && yarn util:pg-dump",
+    "setup": "yarn util:load-vars ts-node --files scripts/setup.ts",
+    "codegen": "yarn util:load-vars graphql-codegen --config codegen.yml",
+    " -- internal scripts, do not change -- ": "",
+    "internal:zapatos": "yarn util:load-vars ts-node --files scripts/codegen-zapatos.ts current",
+    "internal:zapatos:committed": "yarn util:load-vars ts-node --files scripts/codegen-zapatos.ts",
+    "internal:migrate-watch": "yarn util:load-vars ts-node --files scripts/migrate-watch.ts",
+    " -- utilities -- ": "",
+    "util:load-vars": "env-cmd -f .env env-cmd -f ../../../.env",
+    "util:token": "yarn util:load-vars mosaic get-access-token -f ./scripts/resources/channel-admin.json",
+    "util:pg-dump": "yarn util:load-vars mosaic pg-dump",
+    "util:define-func-migration": "yarn mosaic generate-define-func-migration"
+  },
+  "dependencies": {
+    "@axinom/mosaic-db-common": "0.37.0-rc.8",
+    "@axinom/mosaic-graphql-common": "0.13.0-rc.8",
+    "@axinom/mosaic-id-guard": "0.32.0-rc.8",
+    "@axinom/mosaic-id-link-be": "0.21.0-rc.8",
+    "@axinom/mosaic-message-bus": "0.27.0-rc.8",
+    "@axinom/mosaic-message-bus-abstractions": "0.14.0-rc.8",
+    "@axinom/mosaic-messages": "0.43.0-rc.8",
+    "@axinom/mosaic-service-common": "0.49.0-rc.8",
+    "@axinom/mosaic-transactional-inbox-outbox": "0.9.0-rc.8",
+    "@faker-js/faker": "^7.6.0",
+    "@graphile-contrib/pg-simplify-inflector": "^6.1.0",
+    "amqplib": "^0.10.3",
+    "async-mutex": "^0.4.0",
```

Comment:
"async-mutex" can be dropped as it's not used anywhere

Reply:
"generate-password" not used as well. Would check other libs too
![image](https://github.com/Axinom/mosaic-media-template/assets/117624390/a16a63b9-6908-4a30-b199-9998c2efe71b)